fv3core/tests/conftest.py

```diff
@@ -17,6 +17,7 @@ def pytest_addoption(parser):
     parser.addoption("--data_path", action="store", default="./")
     parser.addoption("--threshold_overrides_file", action="store", default=None)
     parser.addoption("--compute_grid", action="store_true")
+    parser.addoption("--dperiodic", action="store_true")
```

Comment:
We need a better "how to test doc"... We need better doc, we rely too much on the `makefile` as a magic thing

Reply:
Added description of `--dperiodic` to the test options section of the fv3core readme, but there doesn't seem like a better place for this? Maybe it belongs in a section of the main readme
'

Reply:
Right now I don't think there's a better space. The docs needs to augmented of a "manual", that would make those things better. Good enough for now